x-pack/platform/test/alerting_api_integration/common/plugins/alerts/server/routes.ts

```diff
@@ -942,4 +942,73 @@ export function defineRoutes(
       }
     }
   );
+
+  router.post(
+    {
+      path: '/api/alerting_fixture/_bulk_edit_params',
+      security: {
+        authz: {
+          enabled: false,
+          reason: 'This route is opted out from authorization',
```

Comment:
Question: Can you add some additional context why this route should be opted out from authz?

Reply:
This is a route added in the test plugin just for testing since this new method is not one we want to expose via HTTP API. I can update the reason.